DOMjudge_tag_release.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+. release.env
+
+for i in `git tag | grep 7`; do git tag -d $i; done
+
+DOWNLOAD_RELEASE_SCRIPT=""
+
+notify_channel () {
+    # Optionally notify the Slack channel
+    # that a build has been triggered
+}
+
+test_tarball () {
+    # Happens as current side effect of docker build
```

Comment:
I think it makes more sense to test the tarball even before pushing the release tag: if something is broken, you want to fix it before it is even in the main github repo.